app/src/main/java/com/nextcloud/talk/webrtc/MagicPeerConnectionWrapper.java

```diff
@@ -105,9 +105,12 @@ public MagicPeerConnectionWrapper(PeerConnectionFactory peerConnectionFactory,
         magicSdpObserver = new MagicSdpObserver();
         hasInitiated = sessionId.compareTo(localSession) < 0;
         this.isMCUPublisher = isMCUPublisher;
-        
-        peerConnection = peerConnectionFactory.createPeerConnection(iceServerList, sdpConstraints,
-                new MagicPeerConnectionObserver());
+
+        PeerConnection.RTCConfiguration configuration = new PeerConnection.RTCConfiguration(iceServerList);
+        configuration.sdpSemantics = PeerConnection.SdpSemantics.PLAN_B;
```

Comment:
Plan-B is already non-functional anymore.
You read the transition guide wrong?

Reply:
The version v96.4664.0 of the WebRTC library is based on the latest extended branch [4664](https://webrtc.googlesource.com/src.git/+log/refs/branch-heads/4664). The latest commit on it is seven weeks old. And this date is before the in phase 4 mentioned date of 29 December, 2021.

Without the change in the `MagicPeerConnectionWrapper`, the following error occurs:

```
E/rtc: #
    # Fatal error in: ../../../webrtc/src/pc/peer_connection.cc, line 818
    # last system error: 0
    # Check failed: !IsUnifiedPlan()
    # AddStream is not available with Unified Plan SdpSemantics. Please use AddTrack instead.
A/libc: Fatal signal 6 (SIGABRT), code -1 (SI_QUEUE) in tid 9538 (signaling_threa), pid 9440 (nextcloud.talk2)
```

With `sdpSemantics=PLAN_B` it's currently working.

But you're right: We will have that problem that SDP semantics not longer available in the next update of the WebRTC library. 

So I'll will migrate our code in this PR.